chains/polygon-pos/config/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+use anyhow::Result;
+use rosetta_core::crypto::address::AddressFormat;
+use rosetta_core::crypto::Algorithm;
+use rosetta_core::BlockchainConfig;
+use serde::{Deserialize, Serialize};
+use std::sync::Arc;
+
+pub fn config(network: &str) -> Result<BlockchainConfig> {
+    anyhow::ensure!(network == "dev");
+    Ok(BlockchainConfig {
+        blockchain: "polygon-pos",
+        network: "dev",
+        algorithm: Algorithm::EcdsaRecoverableSecp256k1,
+        address_format: AddressFormat::Eip55,
+        coin: 1, // TODO: What this coin field means? is it the BIP44 id?
```

Comment:
Is it supposed to be the BIP44 id @dvc94ch ?

Reply:
yes. BIP44 coin 1 is universally used for testnets. Testnets don't get their own BIP44 identifier